src/Runner.Worker/GitHubContext.cs

```diff
@@ -69,5 +69,16 @@ public GitHubContext ShallowCopy()
 
             return copy;
         }
+
+        public bool IsPullRequest()
```

Comment:
Modern C# allows to use `out var` and `is` to provide a shorter way of expressing the same:

```csharp
public bool IsPullRequest()
{
    var data = TryGetValue("event_name", out var value) ? value : null;
    return (data is StringContextData s) && (s == "pull_request");
}
```